pkg/pfcpsim/pfcpsim.go

```diff
@@ -209,6 +209,22 @@ func (c *PFCPClient) SendSessionEstablishmentRequest(pdrs []*ieLib.IE, fars []*i
 	return c.sendMsg(estReq)
 }
 
+func (c *PFCPClient) SendSessionModificationRequest(PeerSEID uint64, pdrs []*ieLib.IE, qers []*ieLib.IE ,fars []*ieLib.IE) error {
+	modifyReq := message.NewSessionModificationRequest(
+		0,
+		0,
+		PeerSEID,
+		c.getNextSequenceNumber(),
+		0,
+	)
+
+	modifyReq.CreatePDR = append(modifyReq.CreatePDR, pdrs...)
```

Comment:
Shouldn't this be `modifyReq.UpdatePDR` ?

Reply:
Right. while rolling back I forgot to update these. Thanks for catching this